homeassistant/components/renson/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "renson",
   "name": "Renson",
-  "codeowners": ["@jimmyd-be"],
+  "codeowners": ["@jimmyd-be", "@rmassch"],
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/renson",
   "iot_class": "local_polling",
-  "requirements": ["renson-endura-delta==1.6.0"]
+  "requirements": ["renson-endura-delta==1.6.0", "pyhealthbox3==0.0.20"]
```

Comment:
Is it possible to combine both requirements into one, and can the library than identify the correct device type?
If not, I suggest creating a new integration and using [brands](https://developers.home-assistant.io/docs/creating_integration_brand/) as currently with all the `if config_entry.data[DEVICE_TYPE]` it looked like two integrations are forced into one.